src/svg/couchdb.svg

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg width="100%" height="100%" viewBox="0 0 512 512" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xml:space="preserve" style="fill-rule:evenodd;clip-rule:evenodd;stroke-linejoin:round;stroke-miterlimit:1.41421;"><path d="M215.611,104.168l-66.836,0l0.137,42.724l66.699,-42.724Zm-116.711,60.988l-13.977,-18.912l0,-42.076l-33.264,0l0,76.832l47.241,-15.844Zm362.131,34.125l0,-95.113l-141.357,0l16.525,2.616l8.241,19.151l-34.133,0l0,14.437l-17.114,-14.437l-43.431,0l-45.555,46.312l30.248,39.793l98.411,-37.33l90.308,73.594l49.883,12.196l7.687,19.637l-88.644,7.96l-4.543,-12.375l-11.966,4.687l-49.261,-41.616l-90.562,58.807l26.675,110.232l233.557,0l0,-208.551l-34.969,0Zm-268.321,195.887l-11.957,-4.952l-23.037,-95.472l-16.96,-4.628l-46.006,-69.272l-24.86,0l0,20.19l-10.842,-8.489l0,29.549l-25.466,-10.841l0,-51.162l0.768,-0.81l-18.35,0l0,208.551l179.77,0l-3.06,-12.664Z" style="fill-rule:nonzero;"/></svg>
```

Comment:
You might consider using [their updated logo](https://github.com/apache/couchdb-documentation/commit/eb4eab6211c3be08714a1e46f8922443c5ef60c2) instead.